packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -452,7 +452,15 @@ export const prepareVaultManagerKit = (
               ephemera.storedCollateralQuote = value;
             },
             onRejected() {
-              facets.helper.observeQuoteNotifier();
+              // NOTE: drastic action, if the quoteNotifier fails, we don't know
+              // the value of the asset, nor do we know how long we'll be in
+              // ignorance. Best choice is to disable actions that require
+              // prices and restart when we have a new price. If we restart the
+              // notifier immediately, we'll trigger an infinite loop, so try
+              // to restart each time we get a request.
+
+              // @ts-expect-error reset value
```

Comment:
this should be put into the typedef instead of suppressed.
 line 205:
```js
 *   storedCollateralQuote: PriceQuote | null;
```

Reply:
done